abb_hardware_interface/package.xml

```diff
@@ -17,7 +17,7 @@
   <depend>hardware_interface</depend>
   <depend>pluginlib</depend>
   <depend>rclcpp</depend>
-  <depend>abb_libegm</depend>
+  <depend>abb_egm_rws_managers</depend>
```

Comment:
We still need abb_libegm, right?

(Please alphabetize, too)

Reply:
We no longer use abb_libegm directly, we get that functionality via abb_egm_rws_managers (which depends on abb_libegm)